src/frontend/src/app/AppAuthenticated.tsx

```diff
@@ -45,6 +46,17 @@ const AppAuthenticated: React.FC<AppAuthenticatedProps> = ({ userId, userRole })
   const [moveContent, setMoveContent] = useState(false);
   const { onGuestHomePage } = useHomePageContext();
 
+  const {
+    data: organization,
+    isLoading: organizationIsLoading,
+    isError: organizationIsError,
+    error: organizationError
+  } = useCurrentOrganization();
+
+  if (organizationIsLoading || !organization || organizationIsError) {
```

Comment:
the checks for organizationIsLoading and !organization should be in the loading indicator check on line 60